### PR TITLE
Support Mongo collection filter, Mysql/Mongo Data export to csv

### PR DESCRIPTION
**Pull request by HaitaoDeng**
image
image

### CODE REVIEW

File: lib/module/redis/const.dart [hunk 1]
```diff
@@ -20,7 +20,7 @@
 class RedisConst {
   static const String defaultHost = "localhost";
   static const int defaultPort = 6379;
-  static const String defaultAddr = "localhost:6379";
+  static const String defaultIp = "localhost:6379";
```
Comment (goflutterjava) [inline lib/module/redis/const.dart:23]: defaultIp?

File: lib/ui/redis/redis_page.dart [hunk 2]
```diff
@@ -69,8 +69,8 @@ class _RedisPageState extends State<RedisPage> {
             columns: [
               DataColumn(label: Text('Id')),
               DataColumn(label: Text('Name')),
-              DataColumn(label: Text('Addr')),
-              DataColumn(label: Text('Username')),
+              DataColumn(label: Text('IP')),
+              DataColumn(label: Text('Port')),
```
Comment (goflutterjava) [inline lib/ui/redis/redis_page.dart:73]: redis has auth message too. why delete this?

File: pubspec.lock [hunk 3]
```diff
@@ -5,147 +5,161 @@ packages:
     dependency: transitive
     description:
       name: _fe_analyzer_shared
-      url: "https://pub.dartlang.org"
+      url: "https://pub.flutter-io.cn"
```
Comment (goflutterjava) [inline pubspec.lock:8]: we should commit `pub.dartlang.org` here, instead of `flutter-io.cn`

File: pubspec.yaml [hunk 4]
```diff
@@ -47,9 +47,11 @@ dependencies:
   # The following adds the Cupertino Icons font to your application.
   # Use with the CupertinoIcons class for iOS style icons.
   cupertino_icons: ^1.0.4
+  csv: ^5.0.0
+  file_picker: ^4.4.0
   mongo_dart: ^0.7.4
   mysql1: ^0.19.2
-  redis: ^2.2.0
+  redis_dart: ^0.1.0+4
```
Comment (goflutterjava) [inline pubspec.yaml:54]: need compare two libraies

File: windows/flutter/generated_plugin_registrant.cc [hunk 5]
```diff
@@ -6,6 +6,9 @@
 
 #include "generated_plugin_registrant.h"
 
+#include <window_size/window_size_plugin.h>
 
 void RegisterPlugins(flutter::PluginRegistry* registry) {
+  WindowSizePluginRegisterWithRegistrar(
```
Comment (goflutterjava) [inline windows/flutter/generated_plugin_registrant.cc:12]: what's this line do?

File: windows/flutter/generated_plugins.cmake [hunk 6]
```diff
@@ -3,6 +3,7 @@
 #
 
 list(APPEND FLUTTER_PLUGIN_LIST
+  window_size
```
Comment (goflutterjava) [inline windows/flutter/generated_plugins.cmake:6]: what's this line do?

**Comment by HaitaoDeng** [comment]
https://github.com/google/flutter-desktop-embedding/blob/master/testbed/windows/flutter/generated_plugins.cmake

**Review by goflutterjava — APPROVED** [review]

**Pull request merged**